electrum/lnworker.py

```diff
@@ -3034,16 +3035,22 @@ async def reestablish_peers_and_channels(self):
                 else:
                     await self.taskgroup.spawn(self.reestablish_peer_for_given_channel(chan))
 
-    def current_target_feerate_per_kw(self) -> Optional[int]:
+    def current_target_feerate_per_kw(self, *, has_anchors: bool) -> Optional[int]:
         if self.network.fee_estimates.has_data():
-            feerate_per_kvbyte = self.network.fee_estimates.eta_target_to_fee(FEE_LN_ETA_TARGET)
+            target: int = FEE_LN_MINIMUM_ETA_TARGET if has_anchors else FEE_LN_ETA_TARGET
+            feerate_per_kvbyte = self.network.fee_estimates.eta_target_to_fee(target)
+            if has_anchors:
+                # set a floor of 5 sat/vb to have some safety margin in case the mempool
+                # grows quickly
+                feerate_per_kvbyte = max(feerate_per_kvbyte, 5000)
```

Comment:
self-note: we add the +50% buffer on top of this later, so the effective floor is 7.5 s/b